composer.json

```diff
@@ -7,6 +7,6 @@
         "php": "^7.0",
         "doctrine/orm": "^2.5.11",
         "doctrine/doctrine-bundle": "^1.6.10",
-        "doctrine/doctrine-migrations-bundle": "^1.3"
+        "doctrine/doctrine-migrations-bundle": "^2.0"
```

Comment:
Could/should this be `^1.3|^2.0` instead?

Reply:
Good idea, applied.